walletsetup.go

```diff
@@ -13,7 +13,7 @@ import (
 
 	"github.com/btcsuite/btcd/chaincfg"
 	"github.com/btcsuite/btcd/wire"
-	"github.com/btcsuite/btcutil"
+	"github.com/btcsuite/btcd/btcutil"
```

Comment:
Needs a `make fmt` to re-order those imports after the rename. Weird that the linter didn't pick up on that.